src/main/java/com/sessions/session7/Homework.java

```diff
@@ -0,0 +1,45 @@
+package com.sessions.session7;
+
+import java.util.ArrayList;
+
+public class Homework {
+    public static void main(String[] args) {
+
+        System.out.println("1. Create a shopping list");
+        String[] shoppingList = {"butter", "oil", "milk", "bread", "carrots"};
+        System.out.print("The shopping list: ");
+        for (String item : shoppingList) {
+            System.out.print(item + " ");
+        }
+
+        System.out.println( "\n\n2.Create a wishlist");
+        ArrayList<String> wishList = new ArrayList<>();
```

Comment:
You can also use the List interface when declaring the reference type. for example
List<String> whishList = new Arraylist<>();